internal/cmdinit/cmdinit.go

```diff
@@ -147,21 +151,40 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 	return nil
 }
 
-var manTemplate = `{{.Name}}
-==================================================
+// manTemplate is the content for the automatically generated README.md file.
+// It uses ' instead of ` since golang doesn't allow using ` in a raw string
+// literal. We do a replace on the content before printing.
+var manTemplate = `# {{.Name}}
 
-# NAME
+## Description
+{{.Description}}
 
-  {{.Name}}
+## Usage
 
-# SYNOPSIS
+### Fetch the package
+'kpt pkg get REPO_URI[.git]/PKG_PATH[@VERSION] {{.Name}}'
+Details: https://googlecontainertools.github.io/kpt/reference/pkg/get/
 
-  kubectl apply --recursive -f {{.Name}}
+### Update package from upstream
```

Comment:
Do we need update here ? This is a day n command and not needed for simple example IMO.

Reply:
regardless, looks like a 1:1 improvement over the original content, so yay.

/lgtm

Reply:
I agree updating a package is more of a day n process, but I think it is useful to demonstrate the key commands for leveraging the advantages of kpt.